microsoft-edge/webview2/get-started/winui2.md

```diff
@@ -11,10 +11,23 @@ keywords: WebView2, webview2, WebView, webview, winui apps, winui, edge, CoreWeb
 ---
 # Get started with WebView2 in WinUI 2 apps (public preview)
 
-In this article, get started creating your first WebView2 app in WinUI 2, and learn about the main features of WebView2. For more information about individual WebView2 APIs, navigate to [Microsoft Edge WebView2 API Reference](../webview2-api-reference.md) and then select the WinRT reference links.
+In this article, you will get started creating your first WebView2 app in WinUI 2, and learn about the main features of WebView2. For more information about individual WebView2 APIs, navigate to [Microsoft Edge WebView2 API Reference](../webview2-api-reference.md) and then select the WinRT reference links.
 
 > [!NOTE]
-> The WinUI 2 package depends on a prerelease WebView2 package. For full API compatibility, use a preview browser channel as your runtime such as the Beta, Dev, or Canary channel of Microsoft Edge.
+> The [WinUI 2 package](https://www.nuget.org/packages/Microsoft.UI.Xaml/2.8.0-prerelease.210927001) takes a dependency on a prerelease WebView2 package. For full API compatibility, use a preview browser channel as your runtime such as the Beta, Dev, or Canary channel of Microsoft Edge.
+
+> [!Important]
+> The WebView2 WinUI Control is actively being developed. There are some features that will be missing or broken. Some of these include:
+> * Download UI -- To maintain compatibility, developers should manually disable this feature by intercepting the [download starting event](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2downloadstartingeventargs?view=webview2-winrt-1.0.1054.31).
+> * Autofill UI
+> * File Picker Dialog
+> * Save Screenshot
```

Comment:
I don't think this is an issue is it? It's the host app side of things that needs to make sure they are using UWP functions for saving a file, which they should be doing anyways.